### PR TITLE
feat: add ci workflow

### PR DESCRIPTION
**Pull request by Jeremy-boo**
1. add ci workflow
2. Change the test script to apply to harbor-arm
3. for issue: https://github.com/goharbor/harbor-arm/issues/9

### CODE REVIEW

File: tests/ci/api_common_install.sh [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+sed "s|#   enabled: false|  enabled: true|" -i make/harbor.yml
+sed "s|#   port: 9090|  port: 9090|" -i make/harbor.yml
+sed "s|#   path: /metrics|  path: /metrics|" -i make/harbor.yml
+
+# build
+sudo make compile build prepare COMPILETAG=compile_golangimage GOBUILDTAGS="include_oss include_gcs" BUILDBIN=true NOTARYFLAG=true TRIVYFLAG=true CHARTFLAG=true GEN_TLS=true PULL_BASE_FROM_DOCKERHUB=false
+
+# set the debugging env
+echo "GC_TIME_WINDOW_HOURS=0" | sudo tee -a ./make/common/config/core/env
+
```
Comment (YangJiao0817) [inline tests/ci/api_common_install.sh:68]: @Jeremy-boo Please add the content of this [PR](https://github.com/goharbor/harbor/pull/16077/files) here

Comment (Jeremy-boo) [reply tests/ci/api_common_install.sh:68]: ok. I‘ll add it

Comment (Jeremy-boo) [reply tests/ci/api_common_install.sh:68]: Sorry, I didn’t understand your specific meaning, can you tell me more in detail

Comment (YangJiao0817) [reply tests/ci/api_common_install.sh:68]: This problem has been fixed, you do not need to do anything, thank you

Comment (Jeremy-boo) [reply tests/ci/api_common_install.sh:68]: ok，thank you

**Review by danielfbm — APPROVED** [review]

**Pull request merged**